src/main/java/com/dougnoel/sentinel/steps/TableVerificationSteps.java

```diff
@@ -134,29 +134,35 @@ public static void verifyColumnExists(String tableName, String assertion, String
     }
     
     /**
-     * Verifies a table column has text for the stored value.
+     * Verifies a table column does or does not contain text for the stored value.
      * <p>
      * <b>Gherkin Examples:</b>
      * <ul>
      * <li>I verify the Name column in the user info table contains the same text entered text for the username</li>
      * <li>I verify the Contact column in the provider info table contains the same text used for the phone number field</li>
      * <li>I verify the Airport Code column in the Airports table contains the same text selected for the airport's code RDU</li>
+     * <li>I verify the Airport Code column in the Airports table does not contain the same text selected for the airport's code RDU</li>
      * </ul>
      * @param columnName String Name of the column to verify
      * @param tableName String Name of the table containing the column
```

Comment:
missing `@param`  documentation for the `assertion` parameter

Reply:
Just added it. Waiting on the checks. Had it fail due to a windows test hiccup so rerunning it.